src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/FindOneTableIntegrationTest.java

```diff
@@ -172,20 +209,21 @@ public void findOneDocIdKey() {
                                   "_id": 2,
                                   "desc": "b",
                                   "valueLong": 42,
-                                  "valueDouble": 0.5
+                                  "valueDouble": 0.5,
+                                  "valueBlob": null
```

Comment:
Either missing or explicit `null` would work: this is what triggered NPE before fix.